features/step_definitions/aspects_steps.rb

```diff
@@ -112,6 +116,12 @@ def aspect_dropdown_visible?
   click_aspect_dropdown
 end
 
+When /^I press the share button$/ do
+  # There were issues 'clicking' the share button on mobile
+  click_share_button
+  sleep 1 # Wait a little bit - especially on mobile views shaing dont works every time
```

Comment:
Ideally there's some UI change we can observe and wait for, something like `expect(page).to have_selector("#dropdown.visible")`, did you have any luck with something like this? :)